projects/addon-editor/directives/tiptap-editor/tiptap-editor.service.ts

```diff
@@ -343,4 +344,8 @@ export class TuiTiptapEditorService extends AbstractTuiEditor {
     setFileLink(preview: TuiEditorAttachedFile): void {
         this.editor.commands.setFileLink(preview);
     }
+
+    setYoutubeVideo(options: TuiYoutubeOptions): void {
+        this.editor.commands.setYoutubeVideo(options as any);
```

Comment:
Why different interface?

Reply:
I want to pass not only number of pixels, but percentages `width="100%"`. if you pass that value, it works, but for some reason their interface only supports numbers